toonz/sources/toonzlib/fill.cpp

```diff
@@ -374,18 +384,33 @@ bool fill(const TRasterCM32P &r, const FillParameters &params,
   int paint = params.m_styleId;
   int fillDepth =
       params.m_shiftFill ? params.m_maxFillDepth : params.m_minFillDepth;
+  TRasterCM32P tempRaster;
+  int styleIndex                                 = 4094;
+  int fakeStyleIndex                             = 4095;
+  if (autoCloseDistance < 0.0) autoCloseDistance = AutocloseDistance;
+  if (fillGaps) {
+    tempRaster            = r->clone();
+    TPalette *tempPalette = params.m_palette->clone();
```

Comment:
This line seems to be unnecessary since `tempPalette` is not used in this scope.
I found that OT crashes here as OT estimates `params.m_palette` should not be NULL in `TPalette::clone()` .

Reply:
Totally right.  That line was left over from a previous idea.  I'll take it out.